client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -34,17 +34,17 @@
 
 // Processing Copy Site Styles
 .copy-site.processing-copy {
-	.processing-step, .stepper-loader {
+	.processing-step, .loading {
 		margin: 5vh auto 0;
 		max-width: 660px;
 	}
 
-	.stepper-loader__title {
+	.loading__title {
 		font-size: 3rem;
 		margin-bottom: 24px;
 	}
 
-	.stepper-loader__progress-bar {
+	.loading__progress-bar {
```

Comment:
I think you missed renaming the classes in this file.

Otherwise, the renaming looks good!

Reply:
Great catch! Fixed.